packages/test-helpers/lib/web3.js

```diff
@@ -0,0 +1,130 @@
+module.exports = web3 => {
+
+  const GETH_NODE_ID = 'Geth'
+  const GANACHE_NODE_ID = 'TestRPC'
+
+  const isGeth = () => web3.version.node.includes(GETH_NODE_ID)
+
+  const isGanache = () => web3.version.node.includes(GANACHE_NODE_ID)
+
+  const advanceBlock = async () => {
+    await web3.currentProvider.send({
+      jsonrpc: '2.0',
+      method: 'evm_mine',
+      id: new Date().getTime()
+    })
+    return web3.eth.getBlock('latest').hash
+  }
+
+  const advanceBlocks = async blocks => {
+    for (let i = 0; i < blocks; i++) {
+      await advanceBlock()
+    }
+  }
+
+  const advanceTime = time => {
+    return web3.currentProvider.sendAsync({
+      jsonrpc: '2.0',
+      method: 'evm_increaseTime',
+      params: [time],
+      id: new Date().getTime()
+    })
+   }
+
+   const advanceTimeAndBlock = async time => {
+    await advanceTime(time)
+    await advanceBlock()
+    return web3.eth.getBlock('latest')
+  }
+  
+  const advanceTimeAndBlocksBy = async (time, numOfBlocks) => {
```

Comment:
Are these time/block controlling utilities used by the court tests? We should usually shy away from these types of utilities, because they aren't usable in an environment outside of ganache.

If they are, I would still push for them to be scoped in a different namespace, e.g. `time-control`

Reply:
Yes, see for instance: https://github.com/aragon/aragon-court/blob/master/test/helpers/wrappers/court.js#L66 and https://github.com/aragon/aragon-court/blob/master/test/helpers/lib/blocks.js

Reply:
Made an issue to add these back in! https://github.com/aragon/contract-helpers/issues/46